capstone/src/test/java/com/google/univiz/api/resource/MapsResourceImplTest.java

```diff
@@ -33,113 +34,111 @@
+
+  private static final CollegeData NYU_COLLEGE_DATA =
+      CollegeData.builder()
+          .setId(NYU_COLLEGE_ID)
+          .setName(NYU_NAME)
+          .setCity(NYU_CITY)
+          .setIsMainCampus(NYU_FLAG_MAIN_CAMPUS)
+          .setLatitude(NYU_LATITUDE)
+          .setLongitude(NYU_LONGITUDE)
+          .setCarnegieSizeDegree(CarnegieSizeDegree.getDegree(CARNEGIE_SIZE_DEGREE))
+          .setAdmissionRate(NYU_ADMISSION_RATE)
+          .setAvgSat(NYU_AVG_SAT)
+          .setNumOfUndergrads(NYU_NUM_OF_UNDERGRADS)
+          .setAvgCost(NYU_AVG_COST)
+          .setRatioOfMen(NYU_RATIO_OF_MEN)
+          .setRatioOfWomen(NYU_RATIO_OF_WOMEN)
+          .build();
+
+  private static final CollegeData STANFORD_COLLEGE_DATA =
+      CollegeData.builder()
+          .setId(STANFORD_COLLEGE_ID)
+          .setName(STANDFORD_NAME)
+          .setCity(STANDFORD_CITY)
+          .setIsMainCampus(STANFORD_FLAG_MAIN_CAMPUS)
+          .setLatitude(STANFORD_LATITUDE)
+          .setLongitude(STANFORD_LONGITUDE)
+          .setCarnegieSizeDegree(CarnegieSizeDegree.getDegree(CARNEGIE_SIZE_DEGREE))
+          .setAdmissionRate(STANFORD_ADMISSION_RATE)
+          .setAvgSat(STANFORD_AVG_SAT)
+          .setNumOfUndergrads(STANFORD_NUM_OF_UNDERGRADS)
+          .setAvgCost(STANFORD_AVG_COST)
+          .setRatioOfMen(STANFORD_RATIO_OF_MEN)
+          .setRatioOfWomen(STANFORD_RATIO_OF_WOMEN)
+          .build();
 
   @Before
   public void guiceSetUp() {
     Guice.createInjector(BoundFieldModule.of(this)).injectMembers(this);
   }
 
   @Test
-  public void testMapsResourceImpl() {
-    CollegeId id = CollegeId.create(COLLEGE_ID);
-    List<CollegeId> ids = Lists.newArrayList(id);
+  public void testMapsResourceImpl() throws IOException {
+    List<CollegeId> ids = Lists.newArrayList(NYU_COLLEGE_ID);
     assertThat(ids).hasSize(1);
 
-    CollegeData college =
-        CollegeData.builder()
-            .setId(id)
-            .setName(NAME)
-            .setCity(CITY)
-            .setIsMainCampus(FLAG_MAIN_CAMPUS)
-            .setLatitude(LATITUDE)
-            .setLongitude(LONGITUDE)
-            .setCarnegieSizeDegree(CarnegieSizeDegree.getDegree(CARNEGIE_SIZE_DEGREE))
-            .setAdmissionRate(ADMISSION_RATE)
-            .setAvgSat(AVG_SAT)
-            .setNumOfUndergrads(NUM_OF_UNDERGRADS)
-            .setAvgCost(AVG_COST)
-            .setRatioOfMen(RATIO_OF_MEN)
-            .setRatioOfWomen(RATIO_OF_WOMEN)
-            .build();
-    List<CollegeData> collegesData = Lists.newArrayList(college);
+    List<CollegeData> collegesData = Lists.newArrayList(NYU_COLLEGE_DATA);
     assertThat(collegesData).hasSize(1);
 
     when(collegeDataApi.getCollegesById(ids)).thenReturn(collegesData);
     List<MapsData> mapsData = mapsImpl.getMapsData(ids);
+    assertThat(mapsData).hasSize(1);
 
     MapsData collegeMapData = mapsData.get(0);
     assertThat(collegeMapData.name()).isEqualTo("New York University");
```

Comment:
Instead of "New York University" you can also use NYU_NAME. But if you decided to move the constants into a shared test object class then you can keep the string literal here as is. Or you can move the NYU_NAME into the common place and use it directly here.

Same thing applies with the other two university names and their latitudes.